### PR TITLE
Add versions logs to task response

### PR DESCRIPTION
**Pull request by medamineziraoui**

### CODE REVIEW

**Comment by medamineziraoui** [comment]
## Why ?
We want to add anthe number of imported versions in the log response from export task.

## Links / Ressources
https://github.com/saagie/gradle-saagie-dataops-plugin/issues/257

## Actual status / Investigations
if we have a zip artifact that contain a job exported with use projectsImport then when successful, show only the name of the job or pipeline imported like shown here 
`job=[{id=4708, name=pm Extraction batch PLCs, versions=3}]`

How?
in the response with condition if versions.size() > 0 then add those changes 
```
response.job << [
  versions: versions.size()
]
```

File: src/test/groovy/io/saagie/plugin/tasks/artifacts/ArtifactsImportTaskTest.groovy [hunk 1]
```diff
@@ -31,6 +31,7 @@ class ArtifactsImportTaskTest extends DataOpsGradleTaskSpecification {
             }
         """
 		
+
```
Comment (DecampsRenan) [inline src/test/groovy/io/saagie/plugin/tasks/artifacts/ArtifactsImportTaskTest.groovy:34]: please remove this line ;)

File: src/main/groovy/io/saagie/plugin/dataops/clients/SaagieClient.groovy [hunk 2]
```diff
@@ -1543,6 +1543,12 @@ class SaagieClient {
 					name : newMappedJobData.job.name
 			]
 			
+			if(versions && versions.size() > 0) {
+				response.job << [
+						versions: versions.size()
+				]
+			}
+
```
Comment (RudyBaer) [inline src/main/groovy/io/saagie/plugin/dataops/clients/SaagieClient.groovy:1551]: A mon avis ce bloc de code peut etre intégré dans la condition au dessus
if (versions && versions.size() >= 1) {
voir meme directement dans 

response.job << [
					id   : job.key,
					name : newMappedJobData.job.name
			]

File: src/main/groovy/io/saagie/plugin/dataops/clients/SaagieClient.groovy [hunk 3]
```diff
@@ -1599,6 +1605,12 @@ class SaagieClient {
 					id   : pipeline.key,
 					name : newMappedPipeline.pipeline.name
 			]
+			
+			if(versions && versions.size() > 0) {
+				response.pipeline << [
+				   versions: versions.size()
+				]
+			}
```
Comment (RudyBaer) [inline src/main/groovy/io/saagie/plugin/dataops/clients/SaagieClient.groovy:1613]: meme chose que mon commentaire précédent

**Pull request merged**